algorithms/ar-bsrh/javascript/binary_Search.js

```diff
@@ -0,0 +1,34 @@
+function binary_Search(items, value) {
```

Comment:
Codacy found an issue: [Identifier 'binary_Search' is not in camel case.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6275500)